### PR TITLE
Move manifest generation out of `alpha` commands

### PR DESCRIPTION
**Pull request by pkosiec**
## Description

Changes proposed in this pull request:

- Move manifest generation out of `alpha` commands

> **NOTE:** Documentation check is failing because of `Error: reviewdog: Too many results (annotations) in diff.`

## Testing

1. Check out the PR
1. Build the binary and run sample commands:

```
# for macOS
make build-tool-cli && mv ./bin/capact-darwin-amd64 /usr/local/bin/capact

capact manifest generate
capact manifest generate interface cap.interface.foo.bar
```

## Reason

As there were significant changes in the manifestgen tool, with feature coverage for all manifest kinds, there's no need to still keep it under `alpha` commands.

### CODE REVIEW

File: cmd/cli/cmd/manifest/generate/generate.go [hunk 1]
```diff
@@ -17,10 +19,16 @@ import (
...
+		Use:   "generate",
+		Short: "OCF Manifests generation",
 		Long:  "Subcommand for various manifest generation operations",
 		Args:  cobra.MaximumNArgs(0),
+		Example: heredoc.WithCLIName(`
+			# To generate manifests interactively, run: 
+			<cli> manifest generate
+			# Then, select which manifests kinds you want to generate.
+			# If the Interface is selected, the Type kind toggles
+			# input and output Type generation for a given Interface.`, cli.Name),
```
Comment (pkosiec) [inline cmd/cli/cmd/manifest/generate/generate.go:31]: I've also added an Example description to highlight that there is an interactive command for generating multiple kinds of manifests.

File: cmd/cli/cmd/alpha/alpha.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ func NewCmd() *cobra.Command {
 	}
 
 	cmd.AddCommand(
-		manifestgen.NewCmd(),
+		generate.NewCmd(),
```
Comment (mszostok) [inline cmd/cli/cmd/alpha/alpha.go:18]: it should be removed, as we move it out of `alpha`. Do not forget to regenerated docs

File: cmd/cli/cmd/manifest/generate/attributes/attribute.go [hunk 3]
```diff
@@ -3,9 +3,10 @@ package attributes
 import (
```
Comment (mszostok) [inline cmd/cli/cmd/manifest/generate/attributes/attribute.go:3]: Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 26

File: cmd/cli/cmd/manifest/generate/implementations/terraform.go [hunk 4]
```diff
@@ -3,9 +3,10 @@ package implementations
 import (
```
Comment (mszostok) [inline cmd/cli/cmd/manifest/generate/implementations/terraform.go:3]: Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 26,29,32

File: cmd/cli/cmd/manifest/generate/interfaces/interface.go [hunk 5]
```diff
@@ -3,9 +3,10 @@ package interfaces
 import (
```
Comment (mszostok) [inline cmd/cli/cmd/manifest/generate/interfaces/interface.go:3]: Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 29

File: cmd/cli/cmd/manifest/generate/interfaces/interfacegroup.go [hunk 6]
```diff
@@ -3,9 +3,10 @@ package interfaces
 import (
```
Comment (mszostok) [inline cmd/cli/cmd/manifest/generate/interfaces/interfacegroup.go:3]: Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 26

File: cmd/cli/cmd/manifest/generate/types/type.go [hunk 7]
```diff
@@ -3,9 +3,10 @@ package types
 import (
```
Comment (mszostok) [inline cmd/cli/cmd/manifest/generate/types/type.go:3]: Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 25

**Review by mszostok — APPROVED** [review]

**Pull request merged**